GameServerLib/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -158,6 +158,13 @@ public int GetTeamSize()
 
         public bool CanMove()
         {
+            foreach (var buff in GetBuffs())
+            {
+                if (!buff.OriginSpell.SpellData.CanMoveWhileChanneling && buff.Name != "Recall")
```

Comment:
I think that `recall` shouldn't be special. You need to check if you can should find a solution that moving with recall will simply cancel it. I'm not sure also if we need to check the can move here, or rather each buff will add restrictions on moving.

Maybe add ` && !OriginSpell.SpellData.CantCancelWhileChanneling ` instead of this recall hardcoding.